java/client/src/org/openqa/selenium/remote/RemoteWebElement.java

```diff
@@ -349,13 +349,16 @@ public Dimension getSize() {
     return new Dimension(width, height);
   }
 
+  @SuppressWarnings({"unchecked"})
   public Rectangle getRect() {
     Response response = execute(DriverCommand.GET_ELEMENT_RECT, ImmutableMap.of("id", id));
     Map<String, Object> rawRect = (Map<String, Object>) response.getValue();
-    int x = ((Number) rawRect.get("x")).intValue();
-    int y = ((Number) rawRect.get("y")).intValue();
-    int width = ((Number) rawRect.get("width")).intValue();
-    int height = ((Number) rawRect.get("height")).intValue();
+    Map<String, Object> origin = (Map<String, Object>) rawRect.get("origin");
+    Map<String, Object> size = (Map<String, Object>) rawRect.get("size");
```

Comment:
https://w3c.github.io/webdriver/webdriver-spec.html#get-element-rect

doesn't reference an 'origin' nor a 'size' in the response body. Bullet point 9 in the spec. The original code appears to be doing the correct thing